### DISCUSSION TITLE
log out -  browser-authorization

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#5783] [Q&A]
Hi everyone, I have a problem, in my webapp I had created both the log in and the log out but at this moment the logout can't finish its core and redirect to the home page this is the error that generates me " Redirect Validation failed: https is required." , account logout happens but fails to redirect"

for login and logout I use ""@itwin/browser-authorization": "^0.5.1"

I'm sure that up to version 2.x everything worked, now I made a couple of version jumps, first to version 3.0.2 and then to version 3.7.2 and the logout function no longer works.

i'm local right now the env file reports this
IMJS_AUTH_CLIENT_LOGOUT_URI = http://localhost:3000/logout

https://ims.bentley.com/connect/endsession?id_token_hint=xxxxxxxxxxxxxx&post_logout_redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Flogout

xxxxxxx represents the token without reporting it all

Thanks for the precious help

**Comment by Paolo-lab** [comment]
hello good morning, I was wondering if by chance I have to create the localhost with https, 
this forces me to create a certificate etc. etc.

can anyone help me on this?

**Comment by aruniverse** [comment]
Can you the 1.x version of the browser-auth-client and see if the issue persists?

**Comment by ben-polinsky** [comment]
Hi. Could you try again using a different path, one other than `logout`? This is a known defect and a fix should be in the works. I'll check on the status of that as well.
